src/views/LeadAdd/LeadButtons/ConnectorSelectModal/Content/index.js

```diff
@@ -309,22 +352,22 @@ export default class ConnectorContent extends React.PureComponent {
     render() {
         const {
             connectorLeads = [],
+            connectorLeadsRequest: {
+                pending,
+            },
```

Comment:
We should have a default value for connectorLeadsRequets until new react-rest-request is used